Jenkinsfile

```
@@ -281,18 +281,13 @@ def getuid() {
 // in faster time-to-result for PRs.
 
 String get_priority() {
-    node {
-        checkoutScm()
-        if (cachedCommitPragma(pragma: 'Queue-jump', def_val: false)) {
-	    string p = '2'
-	} else if (env.BRANCH_NAME == 'master') {
-	    string p = '4'
-	} else {
-	    string p = ''
-	}
-	echo "Build priority set to " + p
-	return p
+    if (env.BRANCH_NAME == 'master') {
+        string p = '4'
+    } else {
+        string p = ''
     }
+    echo "Build priority set to " + p
```

```suggestion
    echo "Build priority set to " + p == '' ? 'default' : p,
```

Or maybe even only print anything if `p != ''`.  Either way should avoid a message of `Build priority set to`` (i.e. with no value being displayed).

---

I'm not really comfortable enough in groovy to do this, does it need that , at the end?  If you think it does I'm happy to merge your suggestion.

---

> I'm not really comfortable enough in groovy to do this,

I just copy and pasted (and touched up) your syntax from https://github.com/daos-stack/daos/pull/3955/commits/0a4cbe636a4a2e1863b48ee0df9a2740fee2f0f4#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R443

>does it need that , at the end?

No.  That was a result of my copy-and-paste.  :-)  Good catch.